Barotrauma/BarotraumaClient/ClientSource/Items/Components/Machines/Steering.cs

```diff
@@ -332,7 +332,7 @@ protected override void CreateGUI()
                             Vector2 vel = controlledSub == null ? Vector2.Zero : controlledSub.Velocity;
                             var realWorldVel = ConvertUnits.ToDisplayUnits(vel.X * Physics.DisplayToRealWorldRatio) * 3.6f;
                             if (controlledSub != null && controlledSub.FlippedX) { realWorldVel *= -1; }
-                            return ((int)realWorldVel).ToString();
+                            return MathF.Round(realWorldVel,1).ToString("0.0");
```

Comment:
Why only one decimal? To me it'd seem more consistent to use the same number of decimals on both values.

Reply:
I just wasn't interested in the second decimal I guess. Up to you my main man!

Reply:
![2d](https://github.com/Regalis11/Barotrauma/assets/68995233/b6f05eef-8527-47ab-bd2e-3175e0ba9ef7)
2 digits for the speed display is too much IMO.
I'd rather have 1 digit on both, or 1 digit fir X speed 2 digit for Y.

Reply:
1 digit on both looks very very good

![1d](https://github.com/Regalis11/Barotrauma/assets/68995233/12fe994d-4c2c-4f5e-9c12-84524d8bab70)